Documentation.app/Jekyll/Home.md

```diff
@@ -4,4 +4,22 @@ title: Home
 permalink: /docs/
 ---
 
-# Bow Open API
+# Bow OpenAPI
+
+Bow OpenAPI is a command line tool to generate a network client written in Swift from an OpenAPI specification that uses the environmental effect type `EnvIO` from [Bow](https://bow-swift.io/). Its main features are:
+
+## Parsing of OpenAPI specifications
+
+Bow OpenAPI is able to consume OpenAPI / Swagger specifications as either JSON or YAML files. It relies on the `swagger-codegen` command line tool to parse these files, and then provides a set of templates to customize the output of the tool, together with an additional processing of the result.
+
+## Pure functional Swift code
+
+The generated code is written in Swift using the Bow Effects module, resulting in a 100% functional API. It uses `URLSession` under the hood to perform the network requests.
+
+## Compatible with Swift Package Manager
```

Comment:
For now, we can not offer `bow-openapi` as a library; existing Package is to create the `executable`. In the near future, when we can create library with resources, we can package together with the lib+templates and offer a public API. 

On the other way, we can create a section where explain how to create an `aggregate` target to consume the executable for automating the generation of network client. wdut?

Reply:
I refer to the output of the tool here. What you get as a result is a module that is compatible with SPM. Do you think we need to clarify this?

Reply:
Ah! Great, I think so, because it is a little ambiguous, it seems it is able to work with SPM as a library.